Music import fail

It seems that when we try to import music funkwhale crash. On the browser side we have an error 502.

On server side I've this log:
```
gunicorn[1284]: [2026-05-03 11:30:24 +0200] [1284] [ERROR] Worker (pid:1523) was sent SIGSYS!
```
Note that it seems that the issue is related to the systemd configuration of the service `funkwhale-server`, maybe systemd hardening is a bit too agressive, because the workaround that I found was to lunch funkwhale manually in the console and this way the upload worked as expected.

I've the funkwhale version `2.0.1~ynh5` and Yunohost version `12.1.39`.

--- a/README.md
+++ b/README.md
@@ -12,7 +12,7 @@ Convivial and modern music server
 
 [![🌐 Official app website](https://img.shields.io/badge/Official_app_website-darkgreen?style=for-the-badge)](https://funkwhale.audio/)
 [![App Demo](https://img.shields.io/badge/App_Demo-blue?style=for-the-badge)](https://demo.funkwhale.audio)
-[![Version: 2.0.1~ynh5](https://img.shields.io/badge/Version-2.0.1~ynh5-rgb(18,138,11)?style=for-the-badge)](https://ci-apps.yunohost.org/ci/apps/funkwhale/)
+[![Version: 2.0.1~ynh6](https://img.shields.io/badge/Version-2.0.1~ynh6-rgb(18,138,11)?style=for-the-badge)](https://ci-apps.yunohost.org/ci/apps/funkwhale/)
 
 <div align="center">
 <a href="https://apps.yunohost.org/app/funkwhale"><img height="100px" src="https://github.com/YunoHost/yunohost-artwork/raw/refs/heads/main/badges/neopossum-badges/badge_more_info_on_the_appstore.svg"/></a>

--- a/conf/funkwhale-server.service
+++ b/conf/funkwhale-server.service
@@ -38,7 +38,7 @@ ProtectControlGroups=yes
 ProtectKernelModules=yes
 ProtectKernelTunables=yes
 LockPersonality=yes
-SystemCallArchitectures=native
+#SystemCallArchitectures=native
 SystemCallFilter=~@clock @debug @module @mount @obsolete @reboot @setuid @swap
 
 # Denying access to capabilities that should not be relevant for webapps

--- a/conf/funkwhale-server.service
+++ b/conf/funkwhale-server.service
@@ -39,7 +39,7 @@ ProtectKernelModules=yes
 ProtectKernelTunables=yes
 LockPersonality=yes
 SystemCallArchitectures=native
-SystemCallFilter=~@clock @debug @module @mount @obsolete @reboot @setuid @swap @cpu-emulation @privileged
+SystemCallFilter=~@clock @debug @module @mount @obsolete @reboot @setuid @swap @cpu-emulation
 
 # Denying access to capabilities that should not be relevant for webapps
 # Doc: https://man7.org/linux/man-pages/man7/capabilities.7.html

--- a/conf/funkwhale-server.service
+++ b/conf/funkwhale-server.service
@@ -39,7 +39,7 @@ ProtectKernelModules=yes
 ProtectKernelTunables=yes
 LockPersonality=yes
 SystemCallArchitectures=native
-SystemCallFilter=~@clock @debug @module @mount @obsolete @reboot @setuid @swap @cpu-emulation
+SystemCallFilter=~@clock @debug @module @mount @obsolete @reboot @setuid @swap
 
 # Denying access to capabilities that should not be relevant for webapps
 # Doc: https://man7.org/linux/man-pages/man7/capabilities.7.html

--- a/conf/funkwhale-worker.service
+++ b/conf/funkwhale-worker.service
@@ -36,7 +36,7 @@ ProtectKernelModules=yes
 ProtectKernelTunables=yes
 LockPersonality=yes
 #SystemCallArchitectures=native
-SystemCallFilter=~@clock @debug @module @mount @obsolete @reboot @setuid @swap @cpu-emulation @privileged
+SystemCallFilter=~@clock @debug @module @mount @obsolete @reboot @setuid @swap @cpu-emulation
 
 # Denying access to capabilities that should not be relevant for webapps
 # Doc: https://man7.org/linux/man-pages/man7/capabilities.7.html

--- a/conf/nginx.conf
+++ b/conf/nginx.conf
@@ -7,6 +7,7 @@
     client_max_body_size 100M;
     charset utf-8;
 
+    # headers
     more_set_headers "Content-Security-Policy: default-src 'self'; connect-src https: wss: http: ws: 'self' 'unsafe-eval'; script-src 'self' 'wasm-unsafe-eval'; style-src https: http: 'self' 'unsafe-inline'; img-src https: http: 'self' data:; font-src https: http: 'self' data:; media-src https: http: 'self' data:; object-src 'none'";
     more_set_headers "Referrer-Policy: strict-origin-when-cross-origin";
     more_set_headers "X-Frame-Options: SAMEORIGIN";
@@ -41,7 +42,6 @@
     }
 
     location / {
-        expires 1d;
         include proxy_params_with_auth;
         alias __INSTALL_DIR__/front/dist/;
         try_files $uri $uri/ /index.html;

--- a/doc/PRE_UPGRADE.d/2.0.1~ynh6.md
+++ b/doc/PRE_UPGRADE.d/2.0.1~ynh6.md
@@ -1,0 +1,1 @@
+This is a packaging update which should fix the upload of tracks. If you still encounter issues, don't hesitate to raise an issue on the apps repo or the forum!

--- a/manifest.toml
+++ b/manifest.toml
@@ -7,7 +7,7 @@ description.gl = "Servidor de música moderno e amigable"
 description.ca = "Servidor de música modern i convivial"
 description.fr = "Serveur de musique moderne et convivial"
 
-version = "2.0.1~ynh5"
+version = "2.0.1~ynh6"
 
 maintainers = ["Thovi98"]
 

